src/main/scala/io/iohk/ethereum/blockchain/sync/PeerComparator.scala

```diff
@@ -0,0 +1,9 @@
+package io.iohk.ethereum.blockchain
+
+import io.iohk.ethereum.network.EtcPeerManagerActor.PeerInfo
+
+object PeerComparator {
+
+  def doPeersHaveSameBestBlock(peerInfo1: PeerInfo, peerInfo2: PeerInfo): Boolean =
+    peerInfo1.bestBlockHash == peerInfo2.bestBlockHash
```

Comment:
This looks really simple; wasn't it extracted a bit prematurely?

Reply:
I agree it's too simple to be extracted, but this was explicitly stated on the story during the refinement that the comparison logic should not be a part of the PeersClient. Among other reasons for switching to another comparison logic